src/test/java/com/labuhn/minesweeper/domain/FieldTest.java

```diff
@@ -0,0 +1,22 @@
+package com.labuhn.minesweeper.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+class FieldTest {
+    @Test
+    void shouldBeInitializedAsNotMarked(){
+        Field field = new Field(true,true,0);
+        assertThat(field.getFlagStatus()).isEqualTo(FlagStatus.NOT_MARKED);
+    }
+
+    @Test
+    void shouldBeInitializedWithCorrectValues(){
+        Field field = new Field(true,true,0);
```

Comment:
Please create new Field with one true and one false value, it could be possible that the assignment in the class Field is wrong but this test would not show that